src/resolver.js

```diff
@@ -4,14 +4,17 @@ import {normalizeSwagger} from './helpers'
 
 export function makeFetchJSON(http) {
   return (docPath) => {
-    return http({
+    return Promise.resolve(http({
```

Comment:
Why `Promise.resolve()` is necessary here? Doesn't `http()` always return a promise or is there any case it returns value directly?